magic-modules/make-comment.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+set -x
+
+shopt -s dotglob
```

Comment:
Just curious: why are you tweaking shell's dotglob? Nothing in the file seem to be vulnerable to differences based on the setting.

Reply:
I need to copy the .git folder inside magic modules.

Reply:
Isn't that the shell default state? AFAIK cp -r myrepo/ mycopy/ will copy everything.

Reply:
```
[17:38 test :) ]$ mkdir -p test/foo/.bar
[17:38 test :) ]$ cd test
[17:38 test :) ]$ touch foo/baz
[17:39 test :( ]$ mkdir qux
[17:40 test :) ]$ cp -r foo/* qux/
[17:40 test :) ]$ ls -a qux
.  ..  baz
```

You remember correctly, but there's a twist - it's not default behavior if you're copying using globs, which I need to do here because the output folder already exists, so I can't directly copy the input folder.

Reply:
```
[10:14 test :) ]$ mkdir qux
[10:14 test :) ]$ cp -r foo/ qux/
[10:15 test :) ]$ ls qux/
foo
```

Would be, but it copies into the `qux` folder, which already exists and which I can't delete since it's the concourse-created output folder.  Took me a while to figure out how to do this, but this is what I figured out.